### DISCUSSION TITLE
Should WebAssembly docs be in JavaScript section?

### DISCUSSION THREAD

**Discussion by Josh-Cena** [mdn/mdn-community#151] [Content]
I find it very weird that `WebAssembly` is in the JS docs, because:

- It seems it's not really maintained by the JS team (I, for one, do not understand WASM at all)
- It's not part of the ECMA specs—the [WASM interface spec](https://webassembly.github.io/spec/js-api/#webassembly-namespace) is maintained by W3C
- We do have a dedicated WASM section (and `WebAssembly` already appears in the sidebar of https://developer.mozilla.org/en-US/docs/WebAssembly), and it's awkward when all issues/PRs that really target the WASM interface end up having the "Content:JS" label

Can we move this object to the WASM docs completely? This means removing "/JavaScript/" from the URL and removing it from the JS sidebar.

**Comment by Josh-Cena** [comment]
Pinging @mdn/content-webassembly and @mdn/yari-content-javascript

**Reply by teoli2003** [reply]
cc/ @Rumyra

**Comment by ghost** [comment]
I'm not a member or collaborator, but I agree with the idea here.

> - It's not part of the ECMA specs—the [WASM interface spec](https://webassembly.github.io/spec/js-api/#webassembly-namespace) is maintained by W3C

This in particular makes it really weird to see `WebAssembly` considered to be a part of the "standard" ECMAScript global object, when it's not supposed to be.
Rather, it's global just like the Web APIs.
<https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects#webassembly>

**Comment by Josh-Cena** [comment]
I intend to move all these pages under `WebAssembly/JavaScript_interface`. Pinging once again to get some blessing in doing this:

@Rumyra @mendyberger @mdn/yari-content-javascript

**Reply by Josh-Cena** [reply]
OK, seems I've (silently) got the blessing I need. Went ahead to create https://github.com/mdn/content/pull/20603

**Reply by hamishwillee** [reply]
@Josh-Cena Sorry, did not see this.

IMO WASM is the right place. These are "really" Web APIs, but unlike the others they are intended primarily for toolchain developers not ordinary developers. I think that having them in WASM groups them where they are likely to be discovered by the users who need them.

JavaScript section is definitely not the right place. That's for pure language features. 
I suspect these are in the JavaScript section because WASM is a generic runtime and these are the **JavaScript** APIs to interact with it.

**Reply by Josh-Cena** [reply]
I also thought about web APIs, but ended up putting it in WASM for the reason you said (discoverability). I'm fine with either.
